README.md

```diff
@@ -119,6 +119,7 @@ John is also a [Kubernetes and cloud infrastructure consultant](https://bitfield
 	- [Bytes](#bytes)
 	- [CountLines](#countlines)
 	- [Read](#read)
+	- [Slice](#slice)
```

Comment:
This should come after SHA256Sum() if they're in alphabetical order, shouldn't it?